global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "1.0.0-beta5",
+    "version": "2.0.7",
```

Comment:
This global.json is an artifact from the PCL world, which is needed to build in VS2015 (I think). However, if you use the "dotnet" cmd line, it complains about this SDK. Updated it to a more recent version. This does not have any impact on the libraries.

Reply:
Great